Trappist/src/Promact.Trappist.Core/Controllers/TestSettingsController.cs

```diff
@@ -20,23 +22,37 @@ public TestSettingsController(ITestSettingsRepository settingsRepository)
         /// <param name="id">The parameter "id" is used to get the Settings of a Test by its Id</param>
         /// <returns>Settings saved for the selected Test</returns>
         [HttpGet("{id}")]
-        public IActionResult GetTestSettings([FromRoute] int id)
+        public async Task<IActionResult> GetTestSettings([FromRoute] int id)
         {
-            var settings = _settingsRepository.GetTestSettings(id);
-            return Ok(settings);
+            var settings = await _settingsRepository.GetTestSettingsAsync(id);
+            if (settings == null)
+            {
```

Comment:
Braces can be removed in case 'if' condition has only one line inside it. It will make code compact.